vars/CentOS_7.yml

```diff
@@ -0,0 +1,3 @@
+---
+__kernel_settings_packages: ["tuned", "python-configobj"]
+__kernel_settings_services: ["tuned"]
```

Comment:
Why do you create new vars files for CentOS_7 and RedHat_7?

Reply:
The package is named `python3-configobj` on every platform except EL7.

Reply:
Oh yes, I am blind